deps/uv/include/uv.h

```diff
@@ -805,10 +806,15 @@ struct uv_tty_s {
 typedef enum {
   /* Initial/normal terminal mode */
   UV_TTY_MODE_NORMAL,
-  /* Raw input mode (On Windows, ENABLE_WINDOW_INPUT is also enabled) */
+  /*
+   * Raw input mode (On Windows, ENABLE_WINDOW_INPUT is also enabled).
+   * May become equivalent to UV_TTY_MODE_RAW_VT in future libuv versions.
+   */
   UV_TTY_MODE_RAW,
   /* Binary-safe I/O mode for IPC (Unix-only) */
-  UV_TTY_MODE_IO
+  UV_TTY_MODE_IO,
+  /* Raw input mode. On Windows ENABLE_VIRTUAL_TERMINAL_INPUT is also set. */
+  UV_TTY_MODE_RAW_VT
```

Comment:
Fyi, this bit is why I care and have told the bot to open this PR. I plan to use this to (unconditionally) replace `UV_TTY_MODE_RAW` usage in Node.js in a follow-up PR, since Node.js's readline implementation already handles the differences between the two, so this would strictly be an improvement and makes e.g. https://github.com/nodejs/node/pull/47150 available on Windows.